src/main/resources/templates/gradle/build.gradle

```diff
@@ -1,5 +1,7 @@
-apply plugin: 'liberty'
-apply plugin: 'war'
+plugins {
+    id 'war'
+    id 'io.openliberty.tools:liberty-gradle-plugin' version '3.3'
```

Comment:
The plugin id should be changed to `io.openliberty.tools.gradle.Liberty`, otherwise Gradle throws an errorr.

```
plugins {
    id 'war'
    id 'io.openliberty.tools.gradle.Liberty' version '3.3'
}
```

The plugins block doesn't like the Maven coordinates since it pulls from the Gradle Plugins Repo using its own identifiers.
The latest version of the plugin will be used if no version is specified.

You can see all of the available versions [here](https://plugins.gradle.org/plugin/io.openliberty.tools.gradle.Liberty).

Reply:
That's good to know, thanks!
Turns out I tested with the correct id taken from plugins.gradle.org but copy pasted the maven coordinates in the template because I wasn't aware of that difference >_<

I left the version hardcoded as 3.3 (the latest currently), same as it was before. Is that correct?

Reply:
Resolved with https://github.com/OpenLiberty/start.openliberty.io/pull/58/commits/69c367e3d8273686212d59d2b48232c785da6651